### PR TITLE
VACMS-4212: Add past events bulk ops view.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #4212 
Per convo with @swirtSJW & @kevwalsh , decision made is to create an events tab with an exposed filter to sort by date. This view (for now) is only visible to administrators. 

## Testing done
Visual / Behat

## Screenshots
image

## QA steps

- As an administrator, go to `admin/content/events/past` and visually verify (by clicking into individual events) that only past items are present.
- Confirm that filters and bulk op archive ops work as expected.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/views.view.content.yml [hunk 1]
```diff
@@ -48,7 +49,6 @@ display:
         type: role
         options:
           role:
-            content_creator_resources_and_support: content_creator_resources_and_support
```
Comment (swirtSJW) [inline config/sync/views.view.content.yml:51]: ?: Seems like an unrelated loss.

File: config/sync/views.view.content.yml [hunk 2]
```diff
@@ -1689,6 +1689,519 @@ display:
         - 'config:field.storage.node.field_administration'
         - 'config:workflow_list'
     deleted: false
+  page_3:
```
Comment (swirtSJW) [inline config/sync/views.view.content.yml:1692]: Minor: This needs a more meaningful machine name.

File: config/sync/views.view.content.yml [hunk 3]
```diff
@@ -1689,6 +1689,519 @@ display:
...
+    display_options:
+      path: admin/content/events/past
+      menu:
+        type: tab
+        title: 'Bulk edit content'
```
Comment (swirtSJW) [inline config/sync/views.view.content.yml:1697]: Major: This tab and title setting seems to be repeated.  (left over from the clone)

I think this is why there is no tab 
image

File: config/sync/views.view.content.yml [hunk 4]
```diff
@@ -1689,6 +1689,519 @@ display:
...
+        description: ''
+        expanded: false
+        parent: system.admin_content
+        weight: 0
+        context: '0'
+        menu_name: admin
+      tab_options:
+        type: normal
+        title: Content
+        description: 'Find and manage content'
```
Comment (swirtSJW) [inline config/sync/views.view.content.yml:1707]: Does not seem accurate.

File: tests/behat/drupal-spec-tool/views.feature [hunk 5]
```diff
@@ -91,6 +91,7 @@ Feature: Views
 | Content | Master | default | Master |
 | Content | All content | page_1 | Page |
 | Content | Bulk edit content | page_2 | Page |
+| Content | Bulk edit past events | page_3 | Page |
```
Comment (swirtSJW) [inline tests/behat/drupal-spec-tool/views.feature:94]: Would need o be edited when we get a more meaningful name.

**Review by swirtSJW — CHANGES_REQUESTED** [review]

File: config/sync/views.view.content.yml [hunk 6]
```diff
@@ -805,6 +806,558 @@ display:
...
+        description: ''
+        expanded: false
+        parent: system.admin_content
+        weight: 100
+        context: '0'
+        menu_name: admin
+      tab_options:
+        type: normal
+        title: Content
+        description: 'Archive past events'
```
Comment (swirtSJW) [inline config/sync/views.view.content.yml:824]: minor:  This description is no longer accurate.

**Review by swirtSJW — APPROVED** [review]
Ignore the one minor thing.  This is good.

File: config/sync/views.view.content.yml (hunk 6 above)
Comment (ethanteague) [reply config/sync/views.view.content.yml:824]: Arghh....apologies, @swirtSJW
